news/2024/2024-10-09-osumania-4k-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,422 @@
+Taking into consideration that this is the United States second world cup victory in a row this year (winning the osu!catch world cup in June), and that the osu! team is notoriously strong with 6 consecutive titles under their belt, will the community bear witness to a world cup Championship hat-trick?
+
+## Mappool showcase
+
+<style>
+    .osu-md--news .audio-player {
+        max-width: inherit;
+    }
+</style>
+
+[**Download the pool here! (199 MB)**](https://osu.ppy.sh/beatmaps/packs/P262)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2257700954&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+At last, we've reached the end. This is the hardest the pools will get, and it clearly shows, with a great example being [Mare Nectaris (Camellia's "Ingenii" Remix)](https://osu.ppy.sh/beatmapsets/2253779#mania/4793983) by [0DZ0](https://osu.ppy.sh/users/28156707) (RC7). Containing over four whole minutes of pure rice, it has just about all patterns you could imagine seeing in a chart of this length. In other words, it's a rice tiebreaker.
+
+The remaining two examples can be shown by the [osu! originals](/wiki/osu!_originals) this pool gave us, which you can listen to in the osu! originals recap, or by [clicking this link](#grand-finals) to direct you to that portion of the news post.
+
+[Ashrount](https://osu.ppy.sh/beatmaps/artists/352) has been known for their tracks in other rhythm games, and this time they're bringing us an incredible track for the HB1 slot, [GALVANIZER](https://osu.ppy.sh/beatmapsets/2253764#mania/4793954) mapped by [guden](https://osu.ppy.sh/users/11626065)! As expected of the slot, it's fairly straightforward with 1/4 snaps, but that's not to say it's straightforward to play, as 288BPM is really pushing the limits for what's doable. Doubt it? Give it a try yourself!
+
+To wrap up all mappools, where would we be without the Grand Finals tiebreaker? [kanemiko](https://osu.ppy.sh/beatmaps/artists/367), [Kyutatsuki](https://twitter.com/Kyutatsuki) and [Aoi](https://osu.ppy.sh/beatmaps/artists/183) bring us our final chart, [Fractal Vertex](https://osu.ppy.sh/beatmapsets/2253784#mania/4793996), mapped by [-mint-](https://osu.ppy.sh/users/8976576), [Toaph Daddy](https://osu.ppy.sh/users/7616811), [elexire](https://osu.ppy.sh/users/9206093), [guden](https://osu.ppy.sh/users/11626065), and [MyZterioN-](https://osu.ppy.sh/users/8521723)!
+
+While it might be slightly easier than last year's tiebreaker, that's not to say it is without a doubt the hardest thing in the pool. In typical tiebreaker fashion, it goes through a ton of patterns, relentlessly throwing more and more at you despite how much you'd want a short break. Even if this is out of your skill range, it's heavily recommended to at least give it a look and appreciate the design team that worked hard to make this happen.
```

Comment:
```suggestion
While it might be slightly easier than last year's tiebreaker, that's not to say it is without a doubt the hardest thing in the pool. In typical tiebreaker fashion, it goes through a ton of patterns, relentlessly throwing more and more at you despite how much you'd want a short break. Even if this is out of your skill range, we heavily recommend to at least give it a look and appreciate the work of the designers who toiled away to make this happen.
```